modules/500-openvpn/images/ovpn-admin/werf.inc.yaml

```diff
@@ -0,0 +1,91 @@
+{{- $gitCommit := "dbc48ef3f1141caa7769488dd3273cff530c9df7" }}
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-openvpn-bin-artifact
+fromImage: common/alt
+shell:
+  install:
+    - apt-get update
+    - apt-get install -y openvpn
+    - /binary_replace.sh -i "/usr/sbin/openvpn /bin/bash" -o /relocate
+---
+artifact: {{ .ModuleName }}/ovpn-admin-backend-artifact
+from: {{ .Images.BASE_GOLANG_20_ALPINE }}
+git:
+- add: /{{ $.ModulePath }}/modules/500-{{ $.ModuleName }}/images/{{ $.ImageName }}/patches
+  to: /patches
+  stageDependencies:
+    install:
+      - '**/*'
+mount:
+- fromPath: ~/go-pkg-cache
+  to: /go/pkg
+shell:
+  beforeInstall:
+    - apk add --no-cache git
+  install:
+    - mkdir -p /src
+    - cd /src
+    - git clone https://github.com/flant/ovpn-admin.git .
+    - git checkout {{ $gitCommit }}
+    - find /patches -name '*.patch' -exec git apply {} \;
+    - git status
+    - git diff
+    - echo {{ $gitCommit }} > version
+    - echo "rebuild 123"
```

Comment:
Is it necessary?

Reply:
It's necessary.
>- echo {{ $gitCommit }} > version

The rest has been removed.